modules/vsphere-problem-detector-metrics.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * storage/vsphere-problem-detector-operator.adoc
+
+:operator-name: vSphere Problem Detector Operator
+
+[id="vsphere-problem-detector-operator-metrics_{context}"]
+= Metrics for the vSphere problem detector
+
+The {operator-name} exposes the following metrics for use by the Prometheus-based {product-title} cluster monitoring stack.
```

Comment:
You could remove "Prometheus-based" from this sentence.

Reply:
For consistency with the Monitoring book, you could change "{product-title} cluster monitoring stack" to "{product-title} monitoring stack".